hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/DevSources.java

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.pipeline;
+
+import com.hazelcast.jet.function.SupplierEx;
+
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Easy-to-use sources intended to be used during pipeline development.
+ * It's often convenient to have easy-to-use sources with well defined
+ * behaviour when developing a streaming application.
+ *
+ */
+public final class DevSources {
+    private static final AtomicLong COUNTER = new AtomicLong();
+    private static final String NAME_PREFIX = "dev-source-";
+
+    private DevSources() {
+
+    }
+
+    /**
+     * Streaming source which emit with fixed rate. It attempts to compensate for various system hiccups so the rate
```

Comment:
is there a way to teach Intellij to do this for you?

Reply:
I use [Wrap to Column](https://plugins.jetbrains.com/plugin/7234-wrap-to-column) plugin, i have it configured to wrap at 79 characters. This works for method javadocs, class javadocs will be 4 characters longer, because the configured value includes the leading whitespaces.

For deeper indentations, such as when describing method params, I let marko reformat it when reviewing ;-)